src/core/announcements/validation.test.ts

```diff
@@ -51,29 +52,47 @@ describe("validation", () => {
         const followeeIdentityContract = await new Identity__factory(signer).deploy(followeeAddress);
         await followeeIdentityContract.deployed();
         const followeeTransaction = await register(followeeIdentityContract.address, "George Bluth");
-        followeeId = await getURIFromRegisterTransaction(followeeTransaction);
+        followeeId = serializeToHex(await getURIFromRegisterTransaction(followeeTransaction));
       });
 
       afterAll(async () => {
         await revertHardhat(provider);
       });
 
       it("returns true for valid graph change announcements", async () => {
```

Comment:
does there need to be one test that uses the output of createFollowGraphChange?

Reply:
Yes, but that should be over in the factories test. I'll add one over there.